dev/generate_bom.py

```diff
@@ -101,6 +102,9 @@ def init_argument_parser(cls, parser):
                         help="Output file to write the changelog to.")
     parser.add_argument('--bom_alias', default='',
                         help="Alias to rename the 'real' BOM as. This also sets the Spinnaker version as the alias.")
+    parser.add_argument('--changelog_repo_owner', default='spinnaker',
```

Comment:
This should be the actual source you are using. If you need the owner, ti should be the origin in the git repo you already have. As noted below, this use case doesnt look right.